docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -10,102 +10,126 @@ platforms: ['android', 'ios', 'tvos']
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
-import { ConfigReactNative, ConfigPluginExample } from '~/components/plugins/ConfigSection';
+import { Collapsible } from '~/ui/components/Collapsible';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { GithubIcon } from '@expo/styleguide-icons';
 
-The `expo-updates` library allows you to programmatically control and respond to new updates made available to your app.
+## Installation and Configuration
```

Comment:
```suggestion
## Installation and configuration
```

Whether we decide to keep this here, I'd still suggest not capitalizing the word "configuration" as per our [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#headings)